node/aas-aid/src/asset-interfaces-description.ts

```diff
@@ -1213,12 +1213,15 @@ export class AssetInterfacesDescription {
                 for (const propertyKey in td.properties) {
                     const property: PropertyElement = td.properties[propertyKey];
 
+                    // we get all the property keys and remove known keys by the TD spec at each if clause
+                    let propertyKeys: any = Object.keys(property);
```

Comment:
This propertyKeys handling is the part I find a big ugly but I could not find a better solution. I went initially for a full json-ld processing but it _me_ messed up... So each time the conversion finds a known TD term, it gets removed from the array. In the end, from the unknown terms, the ones with with a `:` are taken for adding their semantic id with some basic json-ld processing